airflow/www/templates/airflow/dags.html

```diff
@@ -256,11 +270,7 @@ <h2>DAGs</h2>
       });
 
       var $input = $(".typeahead");
-      unique_options_search = new Set([
-          {% for token in auto_complete_data %}
-            "{{token}}",
-          {% endfor %}
-        ]);
+      unique_options_search = new Set(getMetaValue('tokens').split(','));
```

Comment:
I would like to rewrite it on Ajax to speed up page load. Not everyone uses the search engine, but everyone has all the data loaded even when there are a lot of them.

Reply:
I created a PR: https://github.com/apache/airflow/pull/4929/files